flocker/node/agents/ebs.py

```diff
@@ -309,6 +311,7 @@ def _enable_boto_logging():
 _enable_boto_logging()
 
 
+@with_cmp(["requested", "discovered"])
```

Comment:
:grey_question: Is it important that comparison works here (just curious, not suggesting any change)?

Also, sad that you couldn't use more of the characteristic library, looks like you basically had to roll your own type checking and __repr__ for other reasons :(.